.github/workflows/ci-additional.yml

```diff
@@ -0,0 +1,79 @@
+name: CI Additional
+
+on: [push, pull_request]
+
+jobs:
+  mindeps:
+    runs-on: "ubuntu-latest"
+    if: "contains(github.event.head_commit.message, 'test-mindeps') || github.event_name != 'pull_request'"
+
+    strategy:
+      matrix:
+        environment: ["mindeps-array-dataframe", "mindeps-bag-delayed"]
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup Conda Environment
+        uses: goanpeca/setup-miniconda@v1.7.0
+        with:
+          miniconda-version: "latest"
+          python-version: "3.7"
+          environment-file: continuous_integration/environment-${{ matrix.environment }}.yaml
+          activate-environment: test-environment
+          auto-activate-base: false
+
+      - name: Install
+        shell: bash -l {0}
+        env:
+          UPSTREAM_DEV: 1
```

Comment:
Previously we weren't testing with `UPSTREAM_DEV` for the mindeps tests. As things can break with development versions of packages, we might consider not setting `UPSTREAM_DEV=1` here